<?xml version="1.0" encoding="UTF-8"?>
<!--

***********************************************************************************
* Copyright (C) 2019 - 2020, BlockSettle AB
* Distributed under the GNU Affero General Public License (AGPL v3)
* See LICENSE or http://www.gnu.org/licenses/agpl.html
*
**********************************************************************************

-->
<ui version="4.0">
 <class>GeneralSettingsPage</class>
 <widget class="QWidget" name="GeneralSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>GENERAL</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="addvancedDialogByDefaultCheckBox">
        <property name="text">
         <string>Use advanced transaction dialog by default</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="subscribeToMdOnStartCheckBox">
        <property name="text">
         <string>Activate Market Data at launch</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="detailedSettlementTxDialogByDefaultCheckBox">
        <property name="text">
         <string>Use detailed settlement dialog by default</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Default wallet</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_defaultWallet"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>30</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>TERMINAL ENCRYPTION</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Manage</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonManage">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Manage</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxLogging">
     <property name="title">
      <string>GENERAL TERMINAL LOGS</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="horizontalSpacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLineEdit" name="logFileName"/>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Minimum log level</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="logLevel">
            <item>
             <property name="text">
              <string>Full Trace</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Debug</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Info</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Warning</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Error</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Critical</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>No Logging</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="chooseLogFileBtn">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="folderButton" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxLoggingMsg">
     <property name="title">
      <string>MATCHING ENGINE LOGS</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="horizontalSpacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLineEdit" name="logMsgFileName"/>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Minimum log level</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="logLevelMsg">
            <property name="currentIndex">
             <number>6</number>
            </property>
            <item>
             <property name="text">
              <string>Full Trace</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Debug</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Info</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Warning</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Error</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Critical</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>No Logging</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="chooseLogFileMsgBtn">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="folderButton" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>SYSTEM TRAY ICON</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="checkBoxMinimizeToTray">
        <property name="text">
         <string>Minimize to tray</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxCloseToTray">
        <property name="text">
         <string>Keep running in background when main window is closed</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxLaunchToTray">
        <property name="text">
         <string>Hide main window on startup</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxShowTxNotification">
        <property name="text">
         <string>Show notifications for non-settlement transactions</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="warnLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Changes will take effect after the application is restarted. </string>
     </property>
     <property name="statusWarningLabel" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>addvancedDialogByDefaultCheckBox</tabstop>
  <tabstop>subscribeToMdOnStartCheckBox</tabstop>
  <tabstop>logFileName</tabstop>
  <tabstop>chooseLogFileBtn</tabstop>
  <tabstop>logLevel</tabstop>
  <tabstop>logMsgFileName</tabstop>
  <tabstop>chooseLogFileMsgBtn</tabstop>
  <tabstop>logLevelMsg</tabstop>
  <tabstop>checkBoxMinimizeToTray</tabstop>
  <tabstop>checkBoxCloseToTray</tabstop>
  <tabstop>checkBoxLaunchToTray</tabstop>
  <tabstop>checkBoxShowTxNotification</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
